index.html

```diff
@@ -16,6 +16,7 @@
 
     <link rel="stylesheet" type="text/css" href="bower_components/semantic/dist/semantic.min.css">
     <link rel="stylesheet" type="text/css" href="css/styles.min.css">
```

Comment:
Remove `<link rel="stylesheet" type="text/css" href="css/styles.min.css">`. I just included it before due to [minifying)(https://developers.google.com/speed/docs/insights/MinifyResources) the css. 

`<link rel="stylesheet" type="text/css" href="css/styles.css">` is the non minified version.